enterprise/server/raft/store/store.go

```diff
@@ -1864,7 +1864,7 @@ func addLocalRangeEdits(oldLeft, newLeft *rfpb.RangeDescriptor, b *rbuilder.Batc
 	if err != nil {
 		return err
 	}
-	b = b.Add(cas)
+	*b = *b.Add(cas)
```

Comment:
I think this can just be `b.Add(cas)`?

The returned object is only returned for chaining -- but it's mutating the underlying object either way.

Reply:
Edit: same below.

Reply:
Ah yeah, fixed